.github/workflows/update_package_properties.yml

```
@@ -0,0 +1,52 @@
+name: Update Package Properties
+
+env:
+  PRJ_NAME: "pg_auto_failover_enterprise"
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+on:
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: "The tag name on the project repo to be released"
+      fancy_version_no:
+        description: "If fancy versioning is needed, set value greater than 1 e.g 2"
+        default: "1"
+      microsoft_email:
+        description: "Email to be written on changelogs"
+        default: "gindibay@microsoft.com"
+      name:
+        description: "Name to be written on changelogs"
+        default: "Gurkan Indibay"
+jobs:
+  update_package_properties:
+    name: Update package properties
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: sudo apt install libcurl4-openssl-dev libssl-dev
+
+      - name: Clone tools branch
+        run: git clone -b v0.8.1 --depth=1 https://github.com/citusdata/tools.git tools
+
+      - name: Set git name and email
+        run: |
+          git config --global user.email "${{ github.event.inputs.microsoft_email }}" && \
+          git config --global user.name "${{ github.event.inputs.name }}"
```

https://github.com/citusdata/packaging/pull/713#discussion_r696468145 :

> Github variables don't need to be in quotation marks. So I don't put quotation marks. For environment variables quotation marks are needed

Does that mean, that you should have the following here:
```suggestion
          git config --global user.email ${{ github.event.inputs.microsoft_email }} && \
          git config --global user.name ${{ github.event.inputs.name }}
```

I do not like or promote your idea on when to use double quotes. I think that we should always use double quote to prevent globbing and word splitting. See: https://github.com/koalaman/shellcheck/wiki/SC2086

For the time being, I am ok with anything as long as you are consistent and have a policy about this.

---

Used double quotes as a standard